lang/go/go.talon

```diff
@@ -117,6 +185,11 @@ state (start | struct | struck):
     insert(" struct {")
     key("enter")
     insert(user.formatted_text(text, "PUBLIC_CAMEL_CASE"))
+type struct <user.text>:
+    insert("type ")
+    insert(user.formatted_text(text, "PUBLIC_CAMEL_CASE"))
+    insert(" struct {")
+    key("enter")
```

Comment:
This appears to be a duplicate of the type struct below with slightly different syntax.  I'd have one, possibly with a `|` to support both syntaxes